### DISCUSSION TITLE
How do I change background color on vim start?

### DISCUSSION THREAD

**Discussion by alexander-drabik** [wezterm/wezterm#4069] [General]
Hi, 
my vim config has a different background color than my wezterm config and it looks a bit bad when vim is running.
I've managed to make a signal, that changes background color to the vim one, but I don't know how to fire it on vim start. Any ideas?

**Comment by wez** [comment]
Take a look through https://wezfurlong.org/wezterm/recipes/passing-data.html

**Comment by vollink** [comment]
VIM has a lot of color support built-in, so it would be best to ask VIM to do its own color assignment (otherwise things like syntax highlight could end up broken).  Start vim, then `:help colo`
